includes/class-give-background-updater.php

```diff
@@ -230,7 +230,7 @@ protected function task( $update ) {
 		try{
 			// Run update.
 			if ( is_array( $update['callback'] ) ) {
-				$update['callback'][0]->$update['callback'][1]();
+				${$update['callback'][0]->$update['callback'][1]}();
```

Comment:
@ravinderk I've implemented the set of code as per the recommendation mentioned in doc http://php.net/manual/en/migration70.incompatible.php. Search for `Indirect access to variables` section. There is an example snippet which explains that it will work both for PHP 5 as well as PHP 7. Also, I've run the Update process after implementing this and not got any issue with it. Let me know if you have any questions.

Reply:
@ravinderk I've updated the code as discussed after testing it properly.